src/types/UiaTextRangeBase.hpp

```diff
@@ -218,6 +221,8 @@ namespace Microsoft::Console::Types
 
         IRawElementProviderSimple* _pProvider;
 
+        std::wstring _wordDelimiters;

```

Comment:
This guy should probably stay an actual `wstring`, but I'd assume most of the other functions should take a ref instead